specialists/agents/loneliness/loneliness_agent.py

```
@@ -289,37 +274,237 @@ async def build_gemini_prompt(
+            # Try to parse if it's a string representation of a list
+            if context.startswith("[") and context.endswith("]"):
+                import ast
+                context_list = ast.literal_eval(context)
+                if isinstance(context_list, list):
+                    formatted_items = []
+                    for item in context_list:
+                        if isinstance(item, dict):
+                            # Format dictionary items nicely
+                            formatted_items.append(f"• {item}")
+                        else:
+                            formatted_items.append(f"• {str(item)}")
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)